criu/servicefd.c

```diff
@@ -312,5 +313,15 @@ int clone_service_fd(struct pstree_item *me)
 	service_fd_id = id;
 	ret = 0;
 
+	next_high_fd = service_fd_base + 1024;
```

Comment:
Where does this magic number 1024 come from? Perhaps I've missed something, but it doesn't seem safe to assume that next_high_fd will be available and won't conflict with other descriptors. I feel like a detailed comment is needed here.